ui/components/MesheryPerformanceComponent.js

```diff
@@ -143,7 +145,12 @@ class MesheryPerformanceComponent extends React.Component {
       return;
     }
 
-    let err = false;
+    if(!pattern.test(url)) {
+      this.setState({urlError: true});
+      return this.handleError("Please check if the url contains a valid protocol.")();
+    }
+
+    let err = false; 
     let tNum = 0;
```

Comment:
Can we avoid defining pattern at the start of the handleSubmit function, as it may confuse future contributors?